scripts/custom_lint_checks.py

```diff
@@ -143,6 +149,419 @@ def process_module(self, node):
                         'no-break-after-hanging-indent', line=line_num + 1)
 
 
+class DocstringParameterChecker(BaseChecker):
```

Comment:
Reference where this class is from.

Reply:
^ as in add a link to where the code is from.

Reply:
Done.